indico/MaKaC/common/cache.py

```diff
@@ -307,6 +308,15 @@ def restore(cls, value):
 
 
 class CacheClient(object):
+    """This is an abstract class. A cache client provide a simple API to get/set/delete cache entries.
+
+    Implementation must provide the following methods:
```

Comment:
It's usually better to add the method with `raise NotImplementedError` (that way e.g. IDEs will show them as abstract) - but of course the comment doesn't hurt either